lib/ex_webrtc/peer_connection.ex

```diff
@@ -372,6 +380,12 @@ defmodule ExWebRTC.PeerConnection do
     new_transceivers = update_local_transceivers(type, state.transceivers, sdp)
     state = set_description(:local, type, sdp, state)
 
+    demuxer = %{
+      state.demuxer
+      | extensions: SDPUtils.get_extensions(sdp),
+        pt_to_mid: SDPUtils.get_payload_types(sdp)
```

Comment:
I wonder about adding ssrc_to_mid too. Should we then move this to apply_remote_description?

Reply:
I'm going to leave it for now, will do some improvements when moving the demuxing to standalone DTLS process.